operator/buffer/memory.go

```diff
@@ -40,7 +41,7 @@ func NewMemoryBufferConfig() *MemoryBufferConfig {
 // back into memory
 func (c MemoryBufferConfig) Build(context operator.BuildContext, pluginID string) (Buffer, error) {
 	mb := &MemoryBuffer{
-		db:            context.Database,
+		// db:            context.Database,
```

Comment:
Probably remove this?
```suggestion
```

Reply:
This file is going to be revamped entirely in an upcoming ticket before `v2.0.0` release. Since this is going into the v2 branch and not master I didn't want to lose track of how it use to work but rather just comment out parts that not longer work.